todo_app/app.py

```diff
@@ -8,4 +9,13 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+     allitems=get_items()
+     return render_template("index.html",todo=allitems)
+     
+
+
+@app.route('/newitem' , methods=['POST'])
+def newitems():
+    Mynewtitle = request.form["title"] 
```

Comment:
The Python naming convention is snake_case for variable/function names.

So this should be `my_new_title`

And e.g. `allitems` should be `all_items`.

Just makes code easier to read/search when things are consistent.